wcivf/apps/people/templates/people/includes/_person_previous_elections_card.html

```diff
@@ -29,8 +32,16 @@ <h3 class="ds-padded">{% trans "Previous Elections" %}</h3>
                         {% else %}
                             <td>{% if person_post.votes_cast %}{% blocktrans with num_votes=person_post.votes_cast|intcomma %}{{ num_votes }} votes (not elected){% endblocktrans %}{% else %}{% trans "Vote count not available" %}{% endif %}</td>
                         {% endif %}
+                        {% if object.previous_party_count %}
```

Comment:
`object.previous_party_count` will resolve to True if the current `PersonPost` has a previous party, but this table will only show previous `PersonPosts`. That means you get the column showing with no data in for a person with a current previous affiliation, but no previous one. This is the only case we'll see!

It will also include the intro text when there is a non-current ballot, because we're using it to include `_previous_party_affiliations.html`.

We have two choices:

1. Change the annotation add a new `has_noncurrent_previous_parties` and `has_current_previous_parties`. Then include the intro text based on `has_current_previous_parties`.
2. Change the "Previous Elections" table to include current elections (changing the text), and add the column based on the current logic. We'll also need to check for current in the intro text.

I think 2 might be better. Looking at it now, it's a little odd that we don't include the current candidacies in that table.